modules/apps/roles/roles-admin-test/src/testIntegration/java/com/liferay/roles/admin/internal/security/permission/contributor/test/RegularRoleContributorTest.java

```diff
@@ -0,0 +1,102 @@
+
+import com.liferay.arquillian.extension.junit.bridge.junit.Arquillian;
+import com.liferay.portal.kernel.model.Group;
+import com.liferay.portal.kernel.model.GroupConstants;
+import com.liferay.portal.kernel.model.Role;
+import com.liferay.portal.kernel.model.User;
+import com.liferay.portal.kernel.model.role.RoleConstants;
+import com.liferay.portal.kernel.security.permission.PermissionChecker;
+import com.liferay.portal.kernel.security.permission.PermissionCheckerFactory;
+import com.liferay.portal.kernel.security.permission.PermissionCheckerFactoryUtil;
+import com.liferay.portal.kernel.security.permission.PermissionThreadLocal;
+import com.liferay.portal.kernel.service.RoleLocalService;
+import com.liferay.portal.kernel.service.UserLocalServiceUtil;
+import com.liferay.portal.kernel.test.rule.AggregateTestRule;
+import com.liferay.portal.kernel.test.rule.DeleteAfterTestRun;
+import com.liferay.portal.kernel.test.util.GroupTestUtil;
+import com.liferay.portal.kernel.test.util.TestPropsValues;
+import com.liferay.portal.kernel.test.util.UserTestUtil;
+import com.liferay.portal.kernel.util.ArrayUtil;
+import com.liferay.portal.test.rule.FeatureFlag;
+import com.liferay.portal.test.rule.Inject;
+import com.liferay.portal.test.rule.LiferayIntegrationTestRule;
+
+import org.junit.Assert;
+import org.junit.Assume;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * @author Alicia García
+ */
+@RunWith(Arquillian.class)
+public class RegularRoleContributorTest {
+
+	@ClassRule
+	@Rule
+	public static final AggregateTestRule aggregateTestRule =
+		new LiferayIntegrationTestRule();
+
+	@Before
+	public void setUp() throws Exception {
+		_group = GroupTestUtil.addGroup(
+			TestPropsValues.getCompanyId(), TestPropsValues.getUserId(),
+			GroupConstants.DEFAULT_PARENT_GROUP_ID, GroupConstants.CMS);
+	}
+
+	@FeatureFlag("LPD-17564")
+	@Test
+	public void testCMSMemberRoleAssignment() throws Exception {
+		Role role = _roleLocalService.fetchRole(
+			_group.getCompanyId(), RoleConstants.CMS_MEMBER);
+
+		Assume.assumeNotNull(role);
+
+		User user = UserTestUtil.addUser();
+
+		UserLocalServiceUtil.addGroupUsers(
+			_group.getGroupId(), new long[] {user.getUserId()});
+
+		PermissionChecker originalPermissionChecker =
+			PermissionThreadLocal.getPermissionChecker();
+
+		try {
+			PermissionThreadLocal.setPermissionChecker(
+				PermissionCheckerFactoryUtil.create(user));
+
+			PermissionChecker permissionChecker =
+				_permissionCheckerFactory.create(user);
+
+			Assert.assertTrue(
+				ArrayUtil.contains(
+					permissionChecker.getRoleIds(
+						user.getUserId(), _group.getGroupId()),
+					role.getRoleId()));
+		}
+		finally {
+			PermissionThreadLocal.setPermissionChecker(
+				originalPermissionChecker);
+
+			UserLocalServiceUtil.deleteUser(user);
+		}
+	}
+
+	@DeleteAfterTestRun
```

Comment:
Do not delete the group.

Use the GroupTestUtil to it will get or add the CMS group (and it should remain)